### ISSUE TITLE
Bug in Tutrial (using Flux.Tracker: derivative)

### ISSUE THREAD

**Issue by aminya** [FluxML/model-zoo#166]
This code here gives this error:
```julia
using Flux.Tracker: derivative
```
```
ERROR: UndefVarError: derivative not defined
```
Apparently, `derivative` is not exported.
https://github.com/FluxML/model-zoo/blob/97c546c2046a50c9319c5a49c5a13e45bc7d597e/tutorials/60-minute-blitz.jl#L122

In such a beginner tutorial, there is no point in including individual functions one by one!
This code should bring every function:
```julia
using Flux
```

**Comment by aminya** [comment]
OK, this is addressed in https://github.com/FluxML/model-zoo/pull/140 or https://github.com/FluxML/model-zoo/pull/143. But there is no one to merge these pull requests.

**Comment by dhairyagandhi96** [comment]
Closed by #143
